test/shared/functions.js

```diff
@@ -1,50 +1,15 @@
-const { expect } = require('chai');
-
-const constants = require('../../constants');
-
-const sendRawTxn = async (input, sender, ethers, provider) => {
-  const txCount = await provider.getTransactionCount(sender.address);
-  const rawTx = {
-    chainId: network.config.chainId,
-    nonce: ethers.utils.hexlify(txCount),
-    to: input.to,
-    value: input.value || 0x00,
-    gasLimit: ethers.utils.hexlify(3000000),
-    gasPrice: ethers.utils.hexlify(25000000000),
-    data: input.data
-  };
-  const rawTransactionHex = await sender.signTransaction(rawTx);
-  const { hash } = await provider.sendTransaction(rawTransactionHex);
-  return await provider.waitForTransaction(hash);
-};
-
-const checkRawTxnResult = async (input, sender, error) => {
-  let result;
-  if (error)
-    if (network.name === 'hardhat') await expect(sendRawTxn(input, sender, ethers, ethers.provider)).to.be.revertedWith(error);
-    else expect.fail('AssertionError: ' + error);
-  else {
-    result = await sendRawTxn(input, sender, ethers, ethers.provider);
-    expect(result.status).to.equal(1);
-  }
-  return result;
-};
-
-const getRandomInt = (min, max) => {
-  min = Math.ceil(min);
-  max = Math.floor(max);
-  return Math.floor(Math.random() * (max - min) + min);
-};
-
-const returnCurrentTimestamp = async () => {
-  const currentBlockNumber = await provider.getBlockNumber();
-  const block = await provider.getBlock(currentBlockNumber);
-  return block.timestamp;
-};
-
-module.exports = {
-  sendRawTxn,
-  checkRawTxnResult,
-  getRandomInt,
-  returnCurrentTimestamp
-};
```

Comment:
sync and merge your branch with main, as otherwise this pr will erase some of the helper function that have recently been added